### DISCUSSION TITLE
Blocky logs checking on docker is too long

### DISCUSSION THREAD

**Discussion by schebotar** [0xERR0R/blocky#1029] [Q&A] [answered]
Hi there
The way I check out Blocky logs on docker is 
```sh
docker logs blocky -f
```
I get all logs of blocky for months since first start, so it takes too much time
Is there any way to get just tail of logs?

**Comment by 0xERR0R** [comment, marked as answer]
Yes, you can tail last X lines or log entries sinceva date, see here https://docs.docker.com/engine/reference/commandline/logs/

I would also suggest to limit you Container logs and not to keep them forever. https://docs.docker.com/config/containers/logging/local/
